sqlglot/parser.py

```diff
@@ -2852,11 +2857,11 @@ def _parse_bitwise(self) -> t.Optional[exp.Expression]:
 
         while True:
             if self._match_set(self.BITWISE):
-                this = self.expression(
-                    self.BITWISE[self._prev.token_type],
-                    this=this,
-                    expression=self._parse_term(),
-                )
+                expr_type = self.BITWISE[self._prev.token_type]
```

Comment:
this change isn't necessary, you can just get change the bitwise mapping right?

Reply:
Ah right, good call thanks!